src/PageHeader/PageHeader.examples.stories.tsx

```diff
@@ -209,60 +217,63 @@ export const FilesPage = () => (
           </ActionMenu>
         </PageHeader.ContextAreaActions>
       </PageHeader.ContextArea>
-      <PageHeader.TitleArea>
+      {/* This is a hack for now. Breadcrumbs are not leading visuals but for now we are displaying them as leading action until we find a better solution. */}
```

Comment:
What about a `PageHeader.Breadcrumb` child?

Reply:
It sounds good to me! I'd like to tackle this in a separate PR (possibly in an API design PR) if that is okay because there is no use case at dotcom for now so it is not a blocker for us ✨